server/websocket_manager.py

```
@@ -0,0 +1,63 @@
+
+
+def broadcast_sync(message: dict) -> None:
+    """Fire-and-forget broadcast from a sync (thread-pool) context."""
+    if _main_loop is None or not _main_loop.is_running():
+        return
+    try:
+        asyncio.run_coroutine_threadsafe(
+            manager.broadcast(message), _main_loop
+        ).result(timeout=1.0)
+    except Exception:
+        pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)